### ISSUE TITLE
Varying the number of subjects' performance 

### ISSUE THREAD

**Issue by angerhang** [activityMonitoring/capture24_neurips#3]
Keep the test set fixed but vary the number of subjects in the training sets to see how much better than having 150 is compared to 10-15 which exiting datasets have.
